internal/constants/constants.go

```diff
@@ -78,6 +78,13 @@ Within each layer, a token takes precedence over a username and password combina
 	FlagLoggingPipelineLogType          = "log-type"
 	FlagLoggingPipelineLogRetentionTime = "log-retention-time"
 
+	FlagCDNDistributionFilterDomain  = "filter.domain"
+	FlagCDNDistributionFilterState   = "filter.state"
```

Comment:
This might be a nitpick, but using dots on flags is not standard and might be unexpected by users

I think it does work with Cobra but why not keep them as `--domain` and `--state` like we do with other commands, for consistency sake?

Reply:
we can keep them like that. I've just kept them as they were in the api.